lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -768,6 +768,13 @@ def detect_platform
             return Platform::EC2
           end
         end
+        open('http://' + METADATA_SERVICE_ADDR +
+                '/latest/meta-data/services/domain') do |f|
```

Comment:
Should we just be checking that the `/latest` path doesn't return a 404?

Reply:
Checking just the `/latest` seems so generic to confirm the EC2 platform.
Even Google / Azure can add that path in future.